src/components/SplitPane.jsx

```diff
@@ -0,0 +1,149 @@
+var React = require('react'),
+    classNames = require('classnames'),
+    Pane = require('./Pane.jsx'),
+    Resizer = require('./Resizer.jsx'),
+    style = require('./component-playground.less');
+
+export default React.createClass({
+  displayName: 'SplitPane',
+
+  propTypes: {
+    minSize: React.PropTypes.number,
+    defaultSize: React.PropTypes.oneOfType([
+      React.PropTypes.string,
+      React.PropTypes.number,
+    ]),
+    split: React.PropTypes.string,
+    onChange: React.PropTypes.func
+  },
+
+  getInitialState: function getInitialState() {
+    return {
+      active: false,
+      resized: false
+    };
+  },
+
+  getDefaultProps: function getDefaultProps() {
+    return {
+      minSize: 0
+    };
+  },
+
+  componentDidMount: function componentDidMount() {
+    document.addEventListener('mouseup', this.onMouseUp);
+    document.addEventListener('mousemove', this.onMouseMove);
+    var ref = this.refs.pane1;
+    if (ref && this.props.defaultSize && !this.state.resized) {
+      ref.setState({
+        size: this.props.defaultSize
+      });
+    }
+  },
+
+  componentWillUnmount: function componentWillUnmount() {
+    document.removeEventListener('mouseup', this.onMouseUp);
+    document.removeEventListener('mousemove', this.onMouseMove);
+  },
+
+  onMouseDown: function onMouseDown(event) {
+    var position = this.props.split === 'vertical' ?
+     event.clientX : event.clientY;
+    this.setState({
+      active: true,
+      position: position
+    });
+  },
+
+  onMouseMove: function onMouseMove(event) {
+    if (this.state.active) {
+      var ref = this.refs.pane1;
+      if (ref) {
+        var node = ref.getDOMNode();
+        if (window.getComputedStyle) {
+          var styles = window.getComputedStyle(node);
+          var width = styles.width.replace('px', '');
+          var height = styles.height.replace('px', '');
```

Comment:
Can we use [offsetWidth](https://developer.mozilla.org/en-US/docs/Web/API/element.offsetWidth) here? /cc @valentin-radulescu-hs 

Alternatively (if we keep this solution) we should use [_.parseInt](http://devdocs.io/lodash~3/index#parseInt)